sqldelight-core/src/test/kotlin/com/squareup/sqldelight/core/DatabaseTest.kt

```diff
@@ -83,28 +83,28 @@ class DatabaseTest {
         |import kotlin.Int
         |
         |class Database(
-        |    openHelper: SupportSQLiteOpenHelper,
-        |    internal val test_tableAdapter: Test_table.Adapter,
-        |    internal val test_table2Adapter: Test_table2.Adapter
+        |        openHelper: SupportSQLiteOpenHelper,
+        |        internal val test_tableAdapter: Test_table.Adapter,
+        |        internal val test_table2Adapter: Test_table2.Adapter
```

Comment:
This is a bug that needs reverted

Reply:
In KP

Reply:
cc @Egorand

Reply:
This isn't considered a continuation indent. It's a multiline function definition. The guide was just changed on master and it matches the JetBrains one as well.

Reply:
Got it. I'll merge these changes in sqld and when its fixed it KP itll trickle in

Reply:
So multiline param lists should now use normal indents? Do you have a link to the JB style guide where this is mentioned? Can fix in KP

Reply:
https://github.com/android/kotlin-guides/issues/37